### PR TITLE
Section command summary

### LINKED ISSUE #330: UG: Command summary sectioning
image

### PR DESCRIPTION
**Pull request by eeliana**
- closes #330

### CODE REVIEW

**Review by Chesterwongz — APPROVED** [review]
LGTM

File: docs/UserGuide.md [hunk 1]
```diff
@@ -772,22 +772,38 @@ UX | User Experience - The experience a user has when using the app.
 <div style="page-break-after: always;"></div>
 
 ## Command summary
-This section lists all available commands in TAB, along with examples on how you can use them.
+This section lists all the available commands in TAB, along with examples on how you can use them.
+The commands are categorised into 4 different categories:
+* [Managing Students](#students)
+* [Managing Lessons](#lessons)
+* [Switching between views](#switching-between-views)
```
Comment (lingshanng) [inline docs/UserGuide.md:779]:
```suggestion
* [Switching Between Views](#switching-between-views)
```

File: docs/UserGuide.md [hunk 2]
```diff
@@ -772,22 +772,38 @@ UX | User Experience - The experience a user has when using the app.
...
+### Lessons
+
+Action | Format, Examples
+--------|------------------
 **Add Lesson** | `ladd INDEX [recurring/[END_DATE]] date/dd MMM yyyy time/HHmm-HHmm subject/SUBJECT rates/LESSON_RATES [f/OUTSTANDING_FEES] [hw/HOMEWORK]…​`<br><br> e.g. `ladd 1 recurring/ date/10 Nov 2021 time/1000-1200 subject/Math rates/50`
 **Edit Lesson** | `ledit INDEX LESSON_INDEX [recurring/[END_DATE]] [date/dd MMM yyyy] [time/HHmm-HHmm] [subject/SUBJECT] [rates/LESSON_RATES] [f/OUTSTANDING_FEES] [hw/HOMEWORK]… [cancel/CANCEL_DATE]… [uncancel/UNCANCEL_DATE]…​`
 **Delete Lesson** | `ldelete INDEX LESSON_INDEX`<br><br> e.g.`ldelete 2 1`
 **Pay Lesson** | `paid INDEX LESSON_INDEX amt/AMOUNT_PAID`
+
+### Switching Between Views
```
Comment (Chesterwongz) [inline docs/UserGuide.md:801]:
```suggestion
### Navigating the UI
```

File: docs/UserGuide.md [hunk 3]
```diff
@@ -772,22 +772,38 @@ UX | User Experience - The experience a user has when using the app.
...
+
+Action | Format
+--------|------------------
+**List Students** | `list`
```
Comment (Chesterwongz) [inline docs/UserGuide.md:805]: It would be nice if each of these linked to their respective commands

File: docs/UserGuide.md [hunk 4]
```diff
@@ -772,22 +772,38 @@ UX | User Experience - The experience a user has when using the app.
 <div style="page-break-after: always;"></div>
 
 ## Command summary
-This section lists all available commands in TAB, along with examples on how you can use them.
+This section lists all the available commands in TAB, along with examples on how you can use them.
+The commands are categorised into 4 different categories:
+* [Managing Students](#students)
+* [Managing Lessons](#lessons)
+* [Switching Between Views](#switching-between-views)
```
Comment (Chesterwongz) [inline docs/UserGuide.md:779]:
```suggestion
* [Navigating the UI](#navigating-the-ui)
```

File: docs/UserGuide.md [hunk 5]
```diff
@@ -864,35 +864,58 @@ UX | User Experience - The experience a user has when using the app.
...
-**View Yearly Calendar** | `year`
-**Navigate to Today in Calendar** | `today`
-**Navigate forward in Calendar** | `next`
-**Navigate backward in Calendar** | `back`
-**View Reminders** | `remind`
-**Clear** |`clear`
-**Undo** | `undo`
-**Redo** | `redo`
-**Exit** | `exit`
+## Command summary
```
Comment (lingshanng) [inline docs/UserGuide.md:867]:
```suggestion
## Command Summary
```

File: docs/UserGuide.md [hunk 6]
```diff
@@ -864,35 +864,58 @@ UX | User Experience - The experience a user has when using the app.
...
+This section lists all the available commands in TAB, along with examples on how you can use them.
+The commands are categorised into 4 different categories:
+* [Managing Students](#students)
+* [Managing Lessons](#lessons)
```
Comment (lingshanng) [inline docs/UserGuide.md:871]: Why the Header titles not the same

File: docs/UserGuide.md [hunk 7]
```diff
@@ -864,35 +864,58 @@ UX | User Experience - The experience a user has when using the app.
...
+* [Navigating the UI](#navigating-the-ui)
+* [General](#general)
+<br/>
+
+### Students
+
+Action | Format, Examples
```
Comment (lingshanng) [inline docs/UserGuide.md:878]:
```suggestion
Action | Format & Examples
```

File: docs/UserGuide.md [hunk 8]
```diff
@@ -864,35 +864,58 @@ UX | User Experience - The experience a user has when using the app.
...
+[**List Students**](#listing-all-students-list) | `list`
+[**View Tags**](#viewing-all-tags-tag) | `tag`
+[**View Calendar**](#switching-to-the-calendar-interface-calendar) | `calendar`
+[**View Daily Calendar**](#viewing-your-daily-calendar-day) | `day`
+[**View Weekly Calendar**](#viewing-your-weekly-calendar-week) | `week`
+[**View Monthly Calendar**](#viewing-your-monthly-calendar-month) | `month`
+[**View Yearly Calendar**](#viewing-your-yearly-calendar-year) | `year`
+[**Navigate to Today in Calendar**](#navigate-to-today-in-the-calendar-today) | `today`
+[**Navigate forward in Calendar**](#navigating-forward-in-the-calendar-next) | `next`
+[**Navigate backward in Calendar**](#navigating-backwards-in-the-calendar-back) | `back`
```
Comment (lingshanng) [inline docs/UserGuide.md:907]:
```suggestion
[**Navigate Forward in Calendar**](#navigating-forward-in-the-calendar-next) | `next`
[**Navigate Backward in Calendar**](#navigating-backwards-in-the-calendar-back) | `back`
```

**Review by lingshanng — APPROVED** [review]
LGTM

**Pull request merged**